### PR TITLE
added target-visuals.md

### PR DESCRIPTION
**Pull request by johnyeokhaishiang**

### CODE REVIEW

**Comment by tchyong** [comment]
@johnyeokhaishiang 
For target visuals, threshold play a very important role for user to quick identify the performance from the chart. 
Use threshold in your use case to show case how threshold use for analysis.
Default threshold is not always suitable for all cases.

**Comment by tchyong** [comment]
@johnyeokhaishiang 
> A dataset is generated using Mockaroo.

no required. 
Just explain what is the use case dataset.

**Comment by johnyeokhaishiang** [comment]
@tchyong @izairiabdlatif Done with changes, ready to be reviewed now.

File: visual/target_visuals.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+### Filter
+
+Add filters to restrict your data so that analysis will be focused on data which are shown in the chart.
+
+<br/>
+
+## Use Case for Radial Gauge
+### Bank's Website Weekly Downtime
+The dataset used simulates a bank's website weekly downtime. For example, Bank ABC will have a daily fixed downtime from 12:00 AM to
```
Comment (tchyong) [inline visual/target_visuals.md:63]: Try no to use simulate. 
It is a real world dataset. 
Example: 
A Bank has a online website and daily schedule maintenance for 2 hours from xx to xx. Management team would like to know the performance (KPI) of their IT department team on 
daily schedule maintenance. 

Using the sample dataset, configure the chart with below settings:
Bindings,
ThrehsoldSetting

Analysis: 
....

Comment (johnyeokhaishiang) [reply visual/target_visuals.md]: @tchyong 
- Updated wordings of each use case's explanation
- Added Trending row under Bindings table
- Added Threshold row under Bindings table

File: visual/target_visuals.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+hours per week. An allowance for mistakes is given 1 hour per day, totalling 7 hours per week.
+
+Using the sample data [here](./sample-data/target-visuals/downtime.csv), configure the chart with settings as follows:
+
+|Bindings|Select|
+|---|---|
+|Actual|\[Sum]*actual_downtime*|
+|Target|\[Sum]*estimated_downtime*|
+|Trending|Decreasing Better|
+|Threshold|From 0%, To 100%, To 150%, To 200%|
```
Comment (tchyong) [inline visual/target_visuals.md:74]: this setting is not same with your screenshot result. 
Here 2 range but screenshot is 3 ranges.

Comment (johnyeokhaishiang) [reply visual/target_visuals.md:74]: @tchyong Just checked again, it is correct, 3 colours threshold, "0 to 100", "100 to 150", "150 to 200".

**Comment by johnyeokhaishiang** [comment]
PR closed due to messy commits, replaced by a new PR #99.

**Pull request closed without merging**